src/components/Todo/Todo.js

```diff
@@ -0,0 +1,75 @@
+import React, {useState, useEffect} from 'react';
+import ItemList from '../ItemList/ItemList';
+import InputItem from '../InputItem/InputItem';
+import Footer from '../Footer/Footer';
+import styles from './Todo.module.css';
+
+const Todo = () => {
+	
+	const initialState = {
+		items: [
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)